azurelinuxagent/common/protocol/wire.py

```diff
@@ -1304,6 +1320,70 @@ def parse(self, xml_text):
         return self
 
 
+class RemoteAccess(object):
+    """
+    Object containing information about user accounts
+    """
+    #
+    # <RemoteAccess>
+    #   <Version/>
+    #   <Incarnation/>
+    #    <Users>
+    #       <User>
+    #         <Name/>
+    #         <Password/>
+    #         <Expiration/>
+    #       </User>
+    #     </Users>
+    #   </RemoteAccess>
+    #
+
+    def __init__(self, xml_text):
+        logger.verbose("Load RemoteAccess.xml")
+        self.version = None
+        self.incarnation = None
+        self.user_list = RemoteAccessUsersList()
+
+        self.xml_text = None
+        self.parse(xml_text)
+
+    def parse(self, xml_text):
+        """
+        Parse xml document containing user account information
+        """
+        if xml_text is None or len(xml_text) == 0:
+            return self
```

Comment:
Did you mean to return self? Also below

Reply:
below yes, here no.  Fixed

---
In reply to: [194024128](https://github.com/Azure/WALinuxAgent/pull/1171#discussion_r194024128) [](ancestors = 194024128)